tests/benchmark/bintime/test_timedelta_array.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import annotations
+
+import numpy as np
+import pytest
+from pytest_benchmark.fixture import BenchmarkFixture
+
+import nitypes.bintime as bt
+
+pytestmark = pytest.mark.benchmark
+
+SHORT_LIST: list[bt.TimeDelta] = [bt.TimeDelta(float(value)) for value in np.arange(-2, 2, 0.5)]
+LONG_LIST: list[bt.TimeDelta] = [bt.TimeDelta(float(value)) for value in np.arange(-100, 100, 0.3)]
+
+
+@pytest.mark.benchmark(group="timedelta_array_construct")
```

Comment:
I was hoping pytest-benchmark could analyze how a benchmark scales for different valeus of N, like Google Benchmark and BenchmarkDotNet can, but I don't see how to do this.

You can at least use `@pytest.mark.parametrize` to combine the test functions, though.